toplevel/examples/lwt_toplevel/dune

```diff
@@ -90,6 +90,8 @@
    %{bin:js_of_ocaml}
    compile
    --pretty
+   --enable
+   effects
```

Comment:
We need to enable effects in the embedded compiler as well...

Reply:
I've pushed a fix for this